src/main/java/org/jetbrains/bsp/bazel/commons/Constants.java

```diff
@@ -46,7 +46,6 @@
   public static final String TEST_RULE_TYPE = "test";
 
   public static final String BUILD_FILE_NAME = "BUILD";
-  public static final String WORKSPACE_FILE_NAME = "WORKSPACE";
```

Comment:
why was it moved?

Reply:
okey, this one could stay there - i will rollback it